### PR TITLE
feat: Add `clock-*` icons

### PR DESCRIPTION
**Pull request by wojtekmaj**
https://user-images.githubusercontent.com/5426427/143689259-76f54ca1-ca12-4077-9515-f05dc1e3221c.mov

image

### CODE REVIEW

File: icons/clock-4.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="10" />
+  <polyline points="12 6 12 12 16 14" />
+</svg>
```
Comment (wojtekmaj) [inline icons/clock-4.svg:14]: I'm thinking, this icon is identical to existing Clock icon, but it doesn't make a lot sense to have clock-1, 2, 3 and then 5, 6, 7.

Comment (ericfennis) [reply icons/clock-4.svg:14]: hmm yeah that's true. But replacing `clock` with `clock-4` is not an option.

Comment (wojtekmaj) [reply icons/clock-4.svg:14]: Let's keep it then?

Comment (ericfennis) [reply icons/clock-4.svg:14]: yess I think that's better than removing one or the other

**Pull request merged**